tom_hermes/hermes.py

```diff
@@ -24,59 +25,52 @@
 logger = logging.getLogger(__name__)
 #logger.setLevel(logging.DEBUG)
 
-HERMES_URL = os.getenv('HERMES_BASE_URL', 'http://hermes.lco.global')
+HERMES_URL = settings.HERMES_API_URL if hasattr(settings, 'HERMES_API_URL') else os.getenv('HERMES_BASE_URL', 'http://hermes.lco.global')
 HERMES_API_VERSION = 0
 HERMES_API_URL = f'{HERMES_URL}/api/v{HERMES_API_VERSION}'
 
 
 class HermesQueryForm(GenericQueryForm):
-    topic = forms.MultipleChoiceField(choices=[], required=False, label='Topic')
```

Comment:
I removed the topic stuff since you weren't using it in the form. Did you have plans for it to be in the form?

Reply:
thanks, yes that was left over from the pre-simplified, over-complicated idea of what this Broker should be.